src/test/ui/type-check/assignment-in-if.stderr

```diff
@@ -16,7 +16,7 @@ error[E0308]: mismatched types
 31 |     if (x = x) {
    |        ^^^^^^^
    |        |
-   |        help: did you mean to compare equality? `x == x`
+   |        help: did you mean to compare equality?: `x == x`
```

Comment:
these look... funny to me now. I think it's just the juxtaposition of `?` and `:`... can we end it with something like  "did you mean to compare equality? Consider: `x == x`" instead?